conductor/submitter_maya.py

```diff
@@ -172,6 +172,22 @@ def constructVerbosityArg(self, renderer):
             return "%s %s" % (varbosity_flag, verbosity_level)
 
         return ""
+    
+    def getForcedArguments(self, renderer):        
+        '''
+        Force certain flags for certainer renderers. These flags are likely
+	renderer specific and might not even have an equivalent amongst the
+	various renderers.        
+        '''
+        
+        if renderer == 'arnold':
```

Comment:
It doesn't look like we have a constant for `arnold`, can you create one?

Reply:
I created for all the renderers